plugins/autosuspend.koplugin/main.lua

```diff
@@ -1,12 +1,7 @@
 local Device = require("device")
 
-if not Device:isCervantes() and
-    not Device:isKindle() and
-    not Device:isKobo() and
-    not Device:isRemarkable() and
-    not Device:isSDL() and
-    not Device:isSonyPRSTUX() and
-    not Device:isPocketBook() then
+-- If a device can power off or go into standby, it can also suspend ;).
+if not Device:canSuspend() then
```

Comment:
Hmm: We can only activate this plugin, if `canSuspend()` is true. Why do we check that later again?

Reply:
I suspect something from an earlier iteration of the PR. (Yes, please remove the second one again. ^_^)

Reply:
On the one hand, it makes the logic clearer (and mirrors the other _enabled* calls), on the other, eh, I get what you mean... ^^.